src/Memory/MemoryUtils.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_MEMORY_UTILS_H_
+#define ORBIT_MEMORY_UTILS_H_
+
+#include "capture.pb.h"
+
+std::optional<orbit_grpc_protos::TotalMemoryUsageInfo> ParseMemInfo(
```

Comment:
I guess we want to use a namespace around those.

Reply:
Thanks for pointing out this mistake. I added the namespace in the commit *[[fixup] Rename MemoryUtils as MemoryTracingUtils and add namespace](https://github.com/google/orbit/pull/1918/commits/8fac00527b70ed498b52a9c6d24757db62b6a7b8)*.